front/src/modules/simulationResult/components/SimulationResultsMap/SimulationResultsMap.tsx

```diff
@@ -90,7 +91,9 @@ const SimulationResultMap = ({
   );
 
   const updateViewportChange = useCallback(
-    (value: Partial<Viewport>) => dispatch(updateViewport(value, undefined)),
+    (value: Partial<Viewport>, { updateRouter } = { updateRouter: false }) => {
+      dispatch(updateOpStudiesViewport(value, `/map`, updateRouter));
+    },
     [dispatch]
   );
```

Comment:
In the previous version, we didn't use the updateRouter value, so we should not use it now.

This would be enough:
- get the updateViewport action from useMapSettingsActions and use it here
- only update the viewport (don't update the url here with updateRouter)
```suggestion
const updateViewportChange = useCallback(
    (value: Partial<Viewport>) => {
      dispatch(updateViewport(value));
    },
    [dispatch]
  );
```